pom.xml

```diff
@@ -123,7 +123,7 @@
     <flume.version>1.6.0</flume.version>
     <zookeeper.version>3.4.6</zookeeper.version>
     <curator.version>2.6.0</curator.version>
-    <hive.group>org.spark-project.hive</hive.group>
+    <hive.group>com.github.hyukjinkwon</hive.group>
```

Comment:
I redistributed Spark's Hive fork that contains single line fix for Hadoop 3 support.